.github/workflows/terraform-docs.yml

```diff
@@ -0,0 +1,39 @@
+name: Generate terraform docs
+
+on:
+  pull_request:
+    paths:
+      - "**/*.tf"
+
+jobs:
+  terraform-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Generate Terraform docs terraform-modules
+        uses: terraform-docs/gh-actions@v1.1.0
+        with:
+          working-dir: terraform-modules
```

Comment:
we don't want to run terraform docs in directories that don't have a tf file. this is one of those instances, but we do want to run it inside of the other directories in terraform-modules. i wonder if there's a way to refactor this a little where we get a list of the directories and then loop through it

Reply:
Thank you, Chelsey! Great idea, will look into it.